### PR TITLE
Updated all "userIsAwayWithMessage" with an {1} tag.

### PR DESCRIPTION
**Pull request by Remski01**
Please note the next:
Few filles or not touched because i dont see how it looks ingame.

### CODE REVIEW

**Comment by Remski01** [comment]
Please dont approve, I see i maked a few small mistakes with the , and . few have . and a few have , 

I'm fix that somewhere tomorrow.

**Comment by Remski01** [comment]
Alright, They all have now the , before the {1} 

Can this please be checked and aproved?

**Comment by mdcfe** [comment]
I believe this wasn't done at the time that `/afk` messages were implemented because they would be unexpected for server owners.

**Comment by Remski01** [comment]
Hello.

Can this be checked and aproved and merged? Or is there an mistake or typo somewhere?

**Comment by montlikadani** [comment]
@DutchUser 
Please add the word "reason" for each message file as the translator translates it later.

**Comment by Remski01** [comment]
Sure, i do that when i can

File: Essentials/src/messages.properties [hunk 1]
```diff
@@ -553,7 +553,7 @@ userdataMoveBackError=Failed to move userdata/{0}.tmp to userdata/{1}\!
 userdataMoveError=Failed to move userdata/{0} to userdata/{1}.tmp\!
 userDoesNotExist=\u00a74The user\u00a7c {0} \u00a74does not exist.
 userIsAway=\u00a77* {0} \u00a77is now AFK.
-userIsAwayWithMessage=\u00a77* {0} \u00a77is now AFK.
+userIsAwayWithMessage=\u00a77* {0} \u00a77is now AFK, reason {1}
```
Comment (mdcfe) [inline Essentials/src/messages.properties]: I'd prefer `... is now AFK. Reason: {1}`, but I guess it's a matter of preference.

**Comment by Ichbinjoe** [comment]
Should be rebased by the author to fit the new formatting style.

**Comment by mdcfe** [comment]
This should be done on Crowdin now.

**Comment by Ichbinjoe** [comment]
Damn it github this has a change!!!!!!

I'm going to manually commit this in as I think I really confused Github :(

**Pull request closed without merging**